idb/postgres/internal/ledger_for_evaluator/ledger_for_evaluator.go

```diff
@@ -388,96 +376,101 @@ func (l *LedgerForEvaluator) loadAccounts(addresses map[basics.Address]struct{})
 		return nil, fmt.Errorf("loadAccounts() err: %w", err)
 	}
 
+	// Add special accounts if needed.
+	for address := range addresses {
+		if l.isSpecialAddress(address) {
+			// The balance of a special address must pass the minimum balance check in
+			// go-algorand's evaluator, so return a sufficiently large balance.
+			var balance uint64 = 1000 * 1000 * 1000 * 1000 * 1000
+			accountData := new(basics.AccountData)
+			*accountData = basics.AccountData{
+				MicroAlgos: basics.MicroAlgos{Raw: balance},
+			}
+			res[address] = accountData
+		}
+	}
+
 	return res, nil
 }
 
-// PreloadAccounts loads the account data for the given addresses and stores them
-// in the internal cache.
-func (l *LedgerForEvaluator) PreloadAccounts(addresses map[basics.Address]struct{}) error {
```

Comment:
Cool. So this is no longer needed because algod uses `LookupWithoutRewards`, which is now has a batch-able interface?